StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+            LocationSelector.FindElement(By.Name("SelectLocation")).Click();
+            Driver.SwitchTo().DefaultContent();
+            System.Threading.Thread.Sleep(1000);
+        }
+         
+        [When(@"I proceed to the payment page")]
+        public void WhenIProceedToThePaymentPage()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.Id("pickUpPointForm")).Displayed);
+            ((IJavaScriptExecutor)Driver).ExecuteScript("arguments[0].scrollIntoView(true);", Driver.FindElement(By.Id("pickUpPointForm")));
+            IWebElement persDetails = Driver.FindElement(By.Id("pickUpPointForm"));
+            IWebElement TitleSelector = persDetails.FindElement(By.Id("pickUpPoint_title_chosen"));
+            TitleSelector.Click();
+            IWebElement TitleSelected = TitleSelector.FindElement(By.ClassName("chosen-drop"));
+            TitleSelected.FindElements(By.ClassName("active-result")).ElementAt(1).Click();
+            //var dropdownpers = Driver.FindElements(By.ClassName("active-result")).Where(x => x.Displayed);
+            //dropdownpers.ElementAt(1).Click();
+            IWebElement Voornaam = Driver.FindElement(By.ClassName("pupFirstName"));
+            Voornaam.FindElement(By.Name("firstName")).SendKeys("Theo");
+            IWebElement Achternaam = Driver.FindElement(By.ClassName("pupLastName"));
+            Achternaam.FindElement(By.Name("lastName")).SendKeys("Test");
+            IWebElement Telefoon = Driver.FindElement(By.ClassName("pupPhone"));
+            Telefoon.FindElement(By.Name("phone1")).SendKeys("0612345678");
+            IWebElement Email = Driver.FindElement(By.ClassName("pupEmail"));
+            Email.FindElement(By.Name("email1")).SendKeys("test@test.test");
+            IWebElement Betalen = Driver.FindElement(By.ClassName("checkoutControls"));
+            Betalen.FindElement(By.Id("continueToPayment")).Click();
+        }
+
+        [When(@"I click the back button in my browser")]
+        public void WhenIClickTheBackButtonInMyBrowser()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("paymentOptionSelect")).Displayed);
+            Driver.Navigate().Back();
+        }
+         
+        [When(@"I change the delivery method to standard")]
+        public void WhenIChangeTheDeliveryMethodToStandard()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("shippingSelection")).Displayed);
+            IWebElement RadioButton2 = Driver.FindElement(By.ClassName("shippingSelection"));
+            RadioButton2.FindElements(By.ClassName("methodName")).ElementAt(1).Click();
+        }
+
+        [Then(@"I should get the option to fill in my own address")]
+        public void ThenIShouldGetTheOptionToFillInMyOwnAddress()
+        {
+            ((IJavaScriptExecutor)Driver).ExecuteScript("arguments[0].scrollIntoView(true);",
+                 Driver.FindElement(By.Id("WC_AddressForm_div_personTitle")));
+            try
```

Comment:
why we need try catch here? 
What exception you are trying to catch? we should not use `Exception`, at least could you please point out what exactly exception gonna be throw here?  

Let's say there is a really exception should be thrown here, then you should put this exception gonna be throw by function, instead of catch it. 

Also, if you are using NUnit,  please add assertion message, otherwise,  your failed message will be `expect true, but was false`.... it will be very confused for Dev/PM/PO to read your report if test failed.